pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -41,6 +44,28 @@ def parse_toml_option(
         except KeyError:
             raise UnrecognizedOption(f"Don't recognize option {opt}") from exc
 
+    if isinstance(action, BooleanOptionalAction):
+        option_strings = action.option_strings
+
+        if not isinstance(value, bool):
+            error_msg = f"{{'{value}'}} {type(value)} is not a supported argument for"
+            error_msg += f"'{opt}', please use either {{true}} or {{false}}."
+            raise ValueError(error_msg)
+
+        if opt.startswith("no") and f"--{opt[3:]}" in option_strings:
+            opposite_opt = opt[3:]
+            val = ["false", "true"][bool(value)]
+            opp_val = ["true", "false"][bool(value)]
```

Comment:
```suggestion
            val = ["false", "true"][value]
            opp_val = ["true", "false"][value]
```

This shouldn't be necessary anymore